indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -301,4 +267,111 @@
             }
         }).isEverythingSelected;
     };
+
+    function _fetchSourceCategory(categoryId) {
+        if (!_categories[categoryId]) {
+            _categories[categoryId] = categoryId;
```

Comment:
It's not unrelated data. The widget accepts also category IDs as valid input and when opening the widget while the request is sent you want to have something to pass. :+1: for root category.